doc/man/s3fs.1

```diff
@@ -401,16 +401,31 @@ For example, "1Y6M10D12h30m30s".
 Most of the generic mount options described in 'man mount' are supported (ro, rw, suid, nosuid, dev, nodev, exec, noexec, atime, noatime, sync async, dirsync).  Filesystems are mounted with '\-onodev,nosuid' by default, which can only be overridden by a privileged user.
 .TP
 There are many FUSE specific mount options that can be specified. e.g. allow_other. See the FUSE README for the full set.
-.SH NOTES
+.SH LOCAL STORAGE CONSUMPTION
 .TP
-The maximum size of objects that s3fs can handle depends on Amazon S3. For example, up to 5 GB when using single PUT API. And up to 5 TB is supported when Multipart Upload API is used.
+s3fs requires local caching for operation. You can enable a local cache with "\-o use_cache" or s3fs uses temporary files to cache pending requests to s3.
+.TP
+Apart from the requirements discussed below, it is recommended to keep enough cache resp. temporary storage to allow one copy each of all files open for reading and writing at any one time.
 .TP
-If enabled via the "use_cache" option, s3fs automatically maintains a local cache of files in the folder specified by use_cache. Whenever s3fs needs to read or write a file on S3, it first downloads the entire file locally to the folder specified by use_cache and operates on it. When fuse_release() is called, s3fs will re-upload the file to S3 if it has been changed. s3fs uses MD5 checksums to minimize downloads from S3.
+.SS Local cache with \[dq]\-o use_cache\[dq]
 .TP
-The folder specified by use_cache is just a local cache. It can be deleted at any time. s3fs rebuilds it on demand.
+s3fs automatically maintains a local cache of files. The cache folder is specified by the parameter of "\-o use_cache". It is only a local cache that can be deleted at any time. s3fs rebuilds it if necessary.
 .TP
 Local file caching works by calculating and comparing MD5 checksums (ETag HTTP header).
 .TP
+Whenever s3fs needs to read or write a file on S3, it first downloads the entire file to the cache directory and operates on it. On closing the file (when fuse_release() is called), s3fs will re-upload the file to S3 if it has been changed. s3fs uses MD5 checksums to minimize downloads from and uploads to S3.
```

Comment:
The description in the original man page was still out of date.
Currently, I think `Whenever s3fs needs to read or write a file on S3, it first creates the file in the cache directory and operates on it.` is better.